services/wallet/history/service_test.go

```diff
@@ -34,10 +35,15 @@ func prepareTestData(data []TestDataPoint) map[uint64][]*DataPoint {
 	return res
 }
 
+// getBlockNumbers returns -1 if block number is nil
```

Comment:
For simplicity, the `*big.Int` in test data is represented by `int64` so we use `-1` to mark `nil` entries which happen now for the data points that don't have the same `BlockID` for an entry.